src/SFA.DAS.EmployerIncentives.Database/Scripts/Generate.MonthEndMetrics.sql

```diff
@@ -33,6 +33,32 @@ SELECT PaymentYear as [(Paste into cell A6) PaymentYear],PaymentPeriod, count(*)
   INNER JOIN incentives.PendingPayment pp ON pp.Id = x.PendingPaymentId
   GROUP By Passed order by passed desc
 
+-- YTD Valid Records (Paste into cell E22)
+;SELECT SUM(Amount) AS [(Paste into cell E22) Valid YTD]
+FROM incentives.Payment
+WHERE PaymentYear = @AcademicYear
+AND PaidDate IS NOT NULL
+
+-- YTD Invalid Records (Paste into cell E23)
+;SELECT SUM(Amount) AS [(Paste into cell E23) Invalid YTD]
+FROM
+(
+SELECT (-1 * SUM(Amount)) AS Amount
+	FROM incentives.Payment
+	WHERE PaymentYear = @AcademicYear
+	AND PaidDate IS NOT NULL
+	UNION ALL
+	SELECT SUM(Amount)
+	FROM incentives.PendingPayment
+	WHERE Id 
+	IN
+	(
+	SELECT DISTINCT PendingPaymentId FROM incentives.PendingPaymentValidationResult
```

Comment:
Shouldn't this be only for failed validation results?

Reply:
The query takes the value of all the pending payments for the academic year that have been validated, and then subtracts the value of all the ones that have passed validation and been paid. The remainder are the ones that have been validated but not paid, so no filter required, I think

Reply:
OK.  Can you add a comment to the source code s it's clear in the future?

Reply:
Done